server/auth/user.ts

```diff
@@ -21,4 +21,7 @@ export interface User {
   selectedTenant?: string;
   credentials?: any;
   proxyCredentials?: any;
+  multitenancy_enabled?:boolean;
+  private_tenant_enabled?:boolean;
+  default_tenant?: string;
```

Comment:
Aren't these settings on node/cluster level? Why are they added to `User` interface?

Reply:
Going forward, this will be useful if we might want to make these settings user level. That's why keeping it here.

Reply:
@abhivka7 How would it work if we make these settings on user level? Would it only support internal users? What about external users such as SAML users?

Reply:
Wouldn't `default_tenant` be the only one of these settings that may be applicable on the user-level? How would it work if some users had multitenancy_enabled, but others don't?